content/chapters/software-stack/lab/quiz/common-functions.md

```diff
@@ -1,52 +1,64 @@
-# printf() System Call
+# Common functions
```

Comment:
```suggestion
# Common Functions
```
In addition, each question should be in a separate file. But it's fine for now. We'll split them another time.